pronto/api/entries.py

```diff
@@ -271,34 +271,42 @@ def get_recent_entries():
           WHERE MC.STATUS = 'Y'
           GROUP BY EM.ENTRY_AC
         ) MC ON E.ENTRY_AC = MC.ENTRY_AC
+        LEFT OUTER JOIN (
+          SELECT ENTRY_AC, 
+          LISTAGG(METHOD_AC, ',') WITHIN GROUP (ORDER BY METHOD_AC) si
+          FROM INTERPRO.ENTRY2METHOD E2M
+          GROUP BY ENTRY_AC
+        ) SIGN ON SIGN.ENTRY_AC=E.ENTRY_AC
```

Comment:
Rename aliases, and do not sort accessions in SQL (done directly in Python):

```suggestion
        LEFT OUTER JOIN (
          SELECT ENTRY_AC, LISTAGG(METHOD_AC, ',') ACCESSIONS
          FROM INTERPRO.ENTRY2METHOD E2M
          GROUP BY ENTRY_AC
        ) EM ON E.ENTRY_AC = EM.ENTRY_AC
```